### DISCUSSION TITLE
No output log with CodeChecker log -b

### DISCUSSION THREAD

**Discussion by junwha** [Ericsson/codechecker#4142] [Q&A]
Hi all,
I'm trying to log build commands of qhull using CodeChecker, but I see no result in output.json
https://github.com/qhull/qhull

`CodeChecker log -b 'make -j`nproc` all'  -o output.json`

The Makefile uses gcc https://github.com/qhull/qhull/blob/master/Makefile#L136
but nothing is shown in the result.

What would be the problem?

Thank you.

**Comment by whisperity** [comment]
Please run the **initial** compilation through the logger. If you have already compiled the project and `make` does not do anything, then the logger will have nothing to log.

**Reply by junwha** [reply]
Thank you for your help. But it does not log the compilation although I firstly build qhull after clone with CodeChecker (same for clean build). The most weird point is that there are compilation commands includes gcc in the build log.

Also I checked my installed version of CodeChecker is working correctly for other projects.

**Comment by bruntib** [comment]
Dear @junwha0511,
Unfortunately I can't reproduce the problem, CodeChecker logger seems to work properly on this qhull project.
Could you add "--verbose debug" flags to the log command?

`CodeChecker log -b make -o output.json --verbose debug`

This creates a file named `codechecker.logger.debug`. Can you see any suspicious logs in this file?
Thank you!
